test/support/fixture_generator.exs

```diff
@@ -0,0 +1,200 @@
+Mix.install([
+  {:membrane_file_plugin, "~> 0.14.0"},
+  {:membrane_hackney_plugin, "~> 0.10.0"},
+  {:membrane_mp4_plugin, "~> 0.25.0"},
+  {:membrane_mp4_format, ">= 0.0.0"},
+  {:membrane_stream_plugin, "~> 0.3.1"},
+  {:membrane_aac_plugin, ">= 0.0.0"},
+  {:membrane_h264_format,
+   git: "https://github.com/membraneframework/membrane_h264_format.git",
+   ref: "ea5a3d2",
```

Comment:
I had problems with using `github` instead of `git` because i was keeping the `.git`, but the ref is here intentionally, as it points to a commit which has both updated H264 stream format and H264.RemoteStream, but that's only temporary until mp4 changes are released